week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/types/index.d.ts

```diff
@@ -0,0 +1,23 @@
+export type TagName = "p" | "div" | "section" | "article";
+
+export interface Root {
+  title: string;
+  subTitle: string;
+}
+
+export interface Price {
+  value: number;
+  currency: string;
+}
+
+export interface Item {
+  name: string;
+  amount: number;
+  unit: string;
+  price: Price;
+}
+
+export interface Cart {
+  heading?: string;
+  items: Item[];
+}
```

Comment:
Rozdzieliłbym elegancko te typy do osobnych plików, tak aby pokazać co jest naszym modelem danych.

Reply:
Poza `Root` i `Cart` - to są prop types